packages/design-system/src/components/Icons/icons.stories.tsx

```diff
@@ -68,6 +69,11 @@ const iconData = [
     component: <BuildingCircleIcon />,
     name: 'BuildingCircleIcon',
   },
+  {
```

Comment:
Did you also want to add the Calendar icon to the gulp doc site in the file `design-system-docs/src/pages/components/Icon/IconComponent.example.jsx` or is it okay to leave out until we migrate?

Reply:
In either case there's a good chance I'll forget to move it over later. I'll add it to the example now and put it on my to-do list to add it to the gatsby one when that's merged